voxceleb/database.yml

```diff
@@ -1,9 +1,9 @@
 Databases:
   VoxCeleb:
-    - VoxCeleb1/dev/wav/{uri}.wav
-    - VoxCeleb1/test/wav/{uri}.wav
-    - VoxCeleb2/dev/aac/{uri}.wav
-    - VoxCeleb2/test/aac/{uri}.wav
+    - /gpfsdswork/dataset/VoxCeleb1/dev/wav/{uri}.wav
+    - /gpfsdswork/dataset/VoxCeleb1/test/wav/{uri}.wav
+    - /gpfsdswork/dataset/VoxCeleb2/dev/aac/{uri}.wav
+    - /gpfsdswork/dataset/VoxCeleb2/test/aac/{uri}.wav
```

Comment:
Those changes are specific to our Jean Zay setup. Please revert.

Reply:
Revert in c63afe502c0c464f5e5cfe3bde85054fcc8fb356